### DISCUSSION TITLE
Increase RAM to 2G instead of 1G

### DISCUSSION THREAD

**Discussion by Schmeckendeugler** [itzg/docker-minecraft-server#2593] [Q&A]
Hi, I'm sort of new to docker & containers, but I get the basic gist of it. What I'd like to to is try and increase the size of RAM for the container & minecraft server so that it uses 2G instead of 1G RAM. If I can figure that out, I may decide to increase to 4 as well.

I thought that maybe I'd edit a docker file; but I cannot find any entry in there pertaining to the actual launch of the java.exe with parameters. 

Aside from that, I've been loving itzg mc docker minecraft server!!

**Comment by itzg** [comment]
That can be changed with an environment variable when creating the container

https://docker-minecraft-server.readthedocs.io/en/latest/configuration/jvm-options/#memory-limit
